packages/meta-harness/tests/test_register.py

```
@@ -0,0 +1,457 @@
+            project=git_project,
+            check=True,
+        )
+        cand_id = json.loads(result.stdout)["cand_id"]
+        manifest = json.loads(
+            (_candidates_dir(git_project) / cand_id / "manifest.json").read_text(encoding="utf-8")
+        )
+        assert manifest["source_commit"] == head
+
+
+def _git(*args: str, cwd: Path) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(["git", *args], cwd=cwd, capture_output=True, text=True, check=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)